src/ui.rs

```diff
@@ -544,7 +546,10 @@ impl std::ops::DerefMut for PofToolsGui {
 impl PofToolsGui {
     pub fn new(display: &Display) -> Self {
         Self {
-            model: Default::default(),
+            model: Box::new(Model {
+                pof_model: pof::Model::default(),
+                texture_map: HashMap::new(),
+            }),
```

Comment:
```suggestion
            model: Default::default(),
```
assuming `derive(Default)` on `Model`